### DISCUSSION TITLE
Washed out colours on wide gamut monitor compared to browser YT

### DISCUSSION THREAD

**Discussion by jiegen78** [FreeTubeApp/FreeTube#4278] [General]
Is it possible to start freetube with some sort of wide gamut profile?
On chrome there is some option to change in about:flags to solve this problem

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Could u try a Chromium based browser and report back how the colors looks like

**Reply by jiegen78** [reply]
do you mean, on normal yt ?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Apologies for the confusion. Go to the YT website in a Chromium browser and check how the colors look like

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
@jiegen78 How did the colors look like on the YT website in a Chromium browser?

**Reply by jiegen78** [reply]
sorry for the big delay. I've used Iron and I confirm colors are washed out, BUT if I go to chrome flags and I search for Force color profile option, and I set SRGB colors are well saturated. I have a wide gamut monitor.

**Reply by jiegen78** [reply]
is there any way to set color profile on freetube?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Im sorry there is nothing we can do about this at the moment. Setting a color profile has been documented in https://github.com/FreeTubeApp/FreeTube/issues/769 so lets talk there about possible implementations
